src/visit_mut.rs

```diff
@@ -176,6 +269,7 @@ impl VisitMut for IdentVisitor {
 
             for segment in segments.iter_mut() {
                 self.replace_identifier_if_mapped(segment);
+                self.visit_path_arguments_mut(&mut segment.arguments);
```

Comment:
What does this do?

Reply:
This is used to visit the generics parameters tested here : https://github.com/exercism/rust-representer/pull/30/files#diff-2e0c4627528e17cff610171950fb5ab1a265e602fdf9e91394a98574251a162f